sdk/identity/azure-identity/src/main/java/com/azure/identity/package-info.java

```diff
@@ -2,7 +2,218 @@
 // Licensed under the MIT License.
 
 /**
- * Package containing various types of credentials and classes
- * for retrieving access tokens from various configurations.
+ * The Azure Identity library provides Azure Active Directory (Azure AD) token authentication support across the Azure SDK.
+ * The library focuses on OAuth authentication with Azure AD, and it offers various credential classes capable of
+ * acquiring an Azure AD token to authenticate service requests. All the credential classes in this package are
+ * implementations of the `TokenCredential` interface offered by azure-core, and any of them can be used to
+ * construct service clients capable of authenticating with a `TokenCredential`.
+ *
+ * <H2> Getting Started</H2>
+ * The `DefaultAzureCredential` is appropriate for most scenarios where the application is intended to ultimately be run in Azure.
+ * This is because the `DefaultAzureCredential` combines credentials commonly used to authenticate when deployed, with credentials
+ * used to authenticate in a development environment. Note, this credential is intended to simplify getting started with the SDK
+ * by handling common scenarios with reasonable default behaviors. Developers who want more control or whose scenario isn't
+ * served by the default settings should use other credential types.
+ * For more information refer to <a href="https://aka.ms/azsdk/java/identity/defaultazurecredential/docs"> Conceptual knowledge and configuration details </a>.
+ *
+ * <p><strong>Sample: Construct a simple DefaultAzureCredential</strong></p>
+ * <!-- src_embed com.azure.identity.credential.defaultazurecredential.construct -->
+ * <pre>
+ * DefaultAzureCredential defaultAzureCredential = new DefaultAzureCredentialBuilder&#40;&#41;
+ *     .build&#40;&#41;;
+ * </pre>
+ * <!-- end com.azure.identity.credential.defaultazurecredential.construct -->
+ *
+ *
+ * <H2> Authenticating on Azure Hosted Platforms via Managed Identity</H2>
```

Comment:
As discussed in our call, it isn't clear to the reader when these other credential types are relevant to the user. We should have an intro section that talks about the existence of these other credentials, and when they should be used over the DAC.